llvm/lib/Target/AMDGPU/Utils/AMDGPUBaseInfo.h

```diff
@@ -14,6 +14,7 @@
 #include "llvm/IR/InstrTypes.h"
 #include "llvm/IR/Module.h"
 #include "llvm/Support/Alignment.h"
+#include "llvm/TargetParser/TargetParser.h"
```

Comment:
Don't change this. It's generally better to forward-declare things in headers, instead of including more headers, to reduce transitive dependencies.

Reply:
I'm calling IsaVersion default constructor in the downstream version of AMDGPUBaseInfo.h in a follow-up patch. Thought I could make it consistent. 
Anyway TargetParser header is needed in AMDGPUWaitCountUtils.h. Should I move this line to AMDGPUWaitCountUtils.h?

Reply:
Done.